yaptide/persistence/models.py

```diff
@@ -189,7 +189,7 @@ class TaskModel(db.Model):
     # currently for Celery one simulation is one job, but in future we might want to split it into multiple tasks
     # example celery job_ids are: 1a2b3c4d-5e6f-7g8h-9i0j-1k2l3m4n5o6p
     # example SLURM job_ids are: 12345678_12
-    task_id: Column[str] = db.Column(db.String, nullable=False, unique=True, doc="Task ID")
+    task_id: Column[str] = db.Column(db.String, nullable=False, doc="Task ID")
```

Comment:
Why this is not unique anymore ?

Reply:
Because task id is unique only within single simulation and that is how it's uniqueness was redefined here: [https://github.com/yaptide/yaptide/blob/651f22c189b6434fd2e06479bf9409334ce10e56/yaptide/persistence/models.py#L213C23-L213C23](url)

Reply:
Good !